textile_auctions/create_auction.sh

```diff
@@ -0,0 +1,77 @@
+num_args=$#
+if [ $num_args -lt 2 ]; then
+	echo "Not enough arguments"
+	echo "Usage: ${0} <dataset name> <api key>"
+	exit 1
+fi
+
+DATASET_NAME=$1
+NGINX_DIR="/data/carfiles/" # where to put the carfile for serving
+API_KEY=$2 # pass this in as second arg, might be convenient to have this as an env variable
+REP_FACTOR="5" # recommended replication factor is 5
+BROKER_URL="https://broker.staging.textile.dev/"
+SERVER_IP="167.71.215.71" # the IP of this NGINX server droplet, this might change later. We can have a subdomain pointing to this in the future.
+
```

Comment:
there should be a logging function that's called to complete a database with at least the following fields

* dataset id ie.,) ds*[0-9]
* dataset size
* description
* specification: BIDS/NWB/.../
* license
* remotes: {}
* web3archive {
    *  filecoin {
        * archived : T/F
        * status: auction/archived/retrieve
        * redundancy: num
        * CAR file: <hash>
        * method: estuary/powergate/bidbot 
        * deal {
            * deal metadata
            ...
         }
      * ipfs {
         * CID:
         * pinned:
         * API metadata
      }
      } 

Will require a review of datalad datasets for design and metadata patterns